### DISCUSSION TITLE
/DEF_SOLID in *INCLUDE_RADIOSS ignored

### DISCUSSION THREAD

**Discussion by Thamos85** [OpenRadioss/OpenRadioss#2917] [Q&A]
im trying to run a human body model (THUMS) under open radioss. To increase the stability i'd like to change the small strain switching option for solids by using /DEF_SOLID in *INCLUDE_RADIOSS. However, everything in /DEF_SOLID is ignored.
I've testes it with a small example and it works, but not with the big THUMS model. Is there any LS DYNA card shadowing /DEF_SOLID?
I've inserted *INCLUDE_RADIOSS directly at the beginning after *KEYWORD
many thanks for any help
Thomas

**Comment by elequiniou** [comment]
Dear Thomas, thanks a lot for your interest in the project. Indeed the THUMS is quite challenging model. The internal team is working on it already and making fast progress, so it would be great to discuss together and hopefully unify our effort ! Please, feel free to reach out to Marian Bulla communitymanager@openradioss.org to initiate discuss if you are interested in such collaboration.
Kind Regards,
Eric
